scripts/prepare-test-cluster.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+set -x
+# Setup to find necessary data from cluster setup
+# Constants
+HTPASSWD_FILE="./htpass"
+USERPASS="developer"
+HTPASSWD_SECRET="htpasswd-secret"
+SETUP_OPERATORS="./scripts/setup-operators.sh"
+# Overrideable information
+DEFAULT_INSTALLER_ASSETS_DIR=${DEFAULT_INSTALLER_ASSETS_DIR:-$(pwd)}
+KUBEADMIN_USER=${KUBEADMIN_USER:-"kubeadmin"}
+KUBEADMIN_PASSWORD_FILE=${KUBEADMIN_PASSWORD_FILE:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeadmin-password"}
+# Default values
+OC_STABLE_LOGIN="false"
```

Comment:
Base on the use of this variable `OC_STABLE_LOGIN`, it seems to me it should be `OC_LOGIN_SUCCEEDED` rather.